shared/common-adapters/avatar.js

```diff
@@ -163,7 +168,8 @@ const mergeProps = (stateProps, dispatchProps, ownProps: OwnProps): PropsWithout
 
   const style = collapseStyles([
     ownProps.style,
-    onClick && platformStyles({isElectron: desktopStyles.clickable}),
+    (onClick || (flags.avatarUploadsEnabled && ownProps.onEditAvatarClick)) &&
```

Comment:
this logic can be in mapDispatchToProps, just make onEditAvatarClick undefined there